addons/account/models/account_report.py

```diff
@@ -495,6 +502,22 @@ def _check_domain_formula(self):
                 raise UserError(_("Invalid domain for expression '%s' of line '%s': %s",
                                 expression.label, expression.report_line_name, expression.formula))
 
+    @api.constrains('formula')
+    def _check_account_code_prefix_formula(self):
```

Comment:
If we do it in 19.0, I'd just avoid creating a distinct function for that, and rename the constraint above to _check_formula, to put both checks in the same constraint.